samples/boards/96b_argonkey/microphone/README.rst

```diff
@@ -0,0 +1,127 @@
+.. _ArgonKey_Microphone:
+
+ArgonKey Board Microphone
+#########################
+
+Overview
+********
+This sample provides an example of how to acquire audio through
+the on-board MP34DT05 microphone. The microphone generates a PDM
+stream which is acquired through I2S. The PDM stream is then
+converted to PCM using the OpenPDMFilter library, which is available
+in source code in this sample.
+
+Requirements
+************
+
+This sample requires the ArgonKey board plus a USB to TTL 1V8 serial
+cable to get the output audio stream. The board can be powered
+in either one of the following two ways:
+
+- mezzanine mode, plugging the ArgonKey to HiKey board thru its 96Board
+  low-speed connector
+- standalone mode, supplying 5V directly on P1 connector
+
+References
+**********
+
+- :ref:`96b_argonkey`
+
+Building and Running
+********************
+
+ .. zephyr-app-commands::
```

Comment:
Sorry, missed this in the previous review.  Directives, such as ``.. zephyr-app-command`` and ``.. code-block`` should be indented at the same level as the paragraphs around them.  For the directive here on line 33, there should be no leading spaces.

Reply:
ok